### PR TITLE
Add troubleshooting guide

### PR DESCRIPTION
**Pull request by fitzthum**
There are probably some typos and inconsistencies. At some point maybe we should move this to its own file, but I think this is fine for now.

In writing this I realized that there are many things we could improve with a few scripts or code changes, such as adding a debug option to Kata that logs the firmware output. I will make some issues after the release.

PTAL @ryansavino 

Signed-off-by: Tobin Feldman-Fitzthum <tobin@ibm.com>

### CODE REVIEW

**Review by sameo — APPROVED** [review]
Nice one, thanks.

File: quickstart.md [hunk 1]
```diff
@@ -434,8 +434,162 @@ root@localhost:/# ls /run/image/
...
+
+To figure out which basic area you problem is in, first make sure that your Kubernetes
+cluster can schedule non-confidential workloads on your worker node. Remove the `kata-*`
+runtime class from your pod yaml and try to run a pod. If your pod still doesn't run,
+please refer to a more general Kubernetes troubleshooting guide.
+
+If your cluster is healthy but you cannot start confidential containers, you might
+be able get some helpful information from Kubernetes.
+Try `kubectl describe pod <your-pod>`
+Sometimes this will give you a useful message pointing to a faied attestation
```
Comment (bpradipt) [inline quickstart.md:453]: nit: faied -> failed

File: quickstart.md [hunk 2]
```diff
@@ -434,8 +434,162 @@ root@localhost:/# ls /run/image/
...
+  pod_annotations = ["io.katacontainers.*"]
+  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata-qemu-sev.options]
+    ConfigPath = "/opt/confidential-containers/share/defaults/kata-containers/configuration-qemu-sev.toml"
+```
+
+The `ConfigPath` entry on the final line shows the path to the Kata configuration file that will be used
+for that runtime class.
+
+While you are looking at the containerd config, find the `[debug]` section near the top and  set `level`
+to `info`. Make sure to restart containerd after editing the containerd config file.
```
Comment (bpradipt) [inline quickstart.md:496]: should the `level` be `debug` ? I think default is `info` in containerd .

File: quickstart.md [hunk 3]
```diff
@@ -434,8 +434,162 @@ root@localhost:/# ls /run/image/
...
+investigate missing dependencies or incorrect configurations.
+
+#### Guest Firmware Logs
+
+If the VM is running but there is no guest output in the log,
+the guest might have stalled in the firmware. Firmware output will
+depend on your firmware and hypervisor. If you are using QEMU and OVMF,
+you can see the OVMF output by adding `-global isa-debugcon.iobase=0x402`
+and `-debugcon file:/tmp/ovmf.log` to the QEMU command line using the
+redirect script described above.
```
Comment (bpradipt) [inline quickstart.md:595]: @fitzthum this is super helpful. Just added some minor comments.

**Review by bpradipt — CHANGES_REQUESTED** [review]
Added few minor comments.
Also a question: should some of the operator debugging guidelines be moved to this section?

File: quickstart.md (hunk 2 above)
Comment (fitzthum) [reply quickstart.md:496]: good thing someone is paying attention

**Comment by fitzthum** [comment]
>  Also a question: should some of the operator debugging guidelines be moved to this section?

Yeah either here or the operator docs. You can see my k8s section is very light. Mainly because I just destroy my cluster and start over anytime it breaks.

Rebased btw

**Review by bpradipt — APPROVED** [review]
/lgtm

**Pull request merged**